supplementary/alica_ros_turtlesim/ros2/CMakeLists.txt

```diff
@@ -31,13 +35,15 @@ endif (NOT Boost_FOUND)
 ###########
 
 include_directories(
+    ../include
```

Comment:
seems weird, what's going on here?

Reply:
This adds the include folder of the parent directory alica_ros_turtlesim, which containts the turtle_interfaces.hpp file. Both ros1 and ros2 have derived classes from it so they need to include it